dc-cudami-client/src/main/java/de/digitalcollections/cudami/client/CudamiBaseClient.java

```diff
@@ -114,6 +124,8 @@ private HttpRequest createPostRequest(String requestUrl) throws JsonProcessingEx
 
   private HttpRequest createPostRequest(String requestUrl, Object bodyObject)
       throws JsonProcessingException {
+    final URI url = createFullUri(requestUrl);
```

Comment:
Die Variable kann auch in Zeile `120` verwendet werden.

Reply:
ja, gefixt